src/_playground/documentation/Header/Header.js

```diff
@@ -5,9 +5,18 @@ export const Header = (props) => {
     const { children } = props;
 
     return (
-        <h1 className='frDocs-Content__header'>
-            {children}
-        </h1>
+        <React.Fragment>
+            <a className='frDocs-Content__githubLogo' href='https://github.com/SAP/fundamental-react'
```

Comment:
If you really wanted to get fancy, you could import the `package.json` file and use `repository.url` for the value in the anchor's `href` attribute.

Reply:
Done!